<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MuokkaaTilikausi</class>
 <widget class="QDialog" name="MuokkaaTilikausi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>160</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Muokkaa tilikautta</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../pic/pic.qrc">
    <normaloff>:/pic/kirja64.png</normaloff>:/pic/kirja64.png</iconset>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QRadioButton" name="poistaRadio">
     <property name="text">
      <string>Poista tilikausi</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QRadioButton" name="paattyyRadio">
     <property name="text">
      <string>Muuta tilikausi p채채ttym채채n</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QDateEdit" name="paattyyDate"/>
   </item>
   <item row="3" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../pic/pic.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>MuokkaaTilikausi</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>133</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>159</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>MuokkaaTilikausi</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>139</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>159</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>paattyyRadio</sender>
   <signal>toggled(bool)</signal>
   <receiver>paattyyDate</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>114</x>
     <y>40</y>
    </hint>
    <hint type="destinationlabel">
     <x>325</x>
     <y>45</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
